spec/create_vm_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require_relative './shared_context'
+
+require 'fog/kubevirt'
+
+describe Fog::Compute do
+  before :all do
+    vcr = KubevirtVCR.new(
+      vcr_directory: 'spec/fixtures/kubevirt/vm',
+      service_class: Fog::Compute
+    )
+    @service = vcr.service
+  end
+
+  it 'creates vm with multipe pvcs' do
+    VCR.use_cassette('vm_create_multi') do
+      begin
+        vm_name = 'test'
+        cpus = 1
+        memory_size = 64
+        pvc = ['mypvc1', 'mypvc2']
```

Comment:
perhaps 'pvcs' since it holds multiple claims.

Reply:
will change